libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController+QrCodeLogin.swift

```diff
@@ -28,44 +28,25 @@
 import Foundation
 
 public extension SalesforceLoginViewController {
-
-    // MARK: - QR Code Login Via UI Bridge API Public Implementation
-
-    /**
-     * Automatically log in with a UI Bridge API login QR code.
-     * - Parameters
-     *     - loginQrCodeContent:  The login QR code content.  This should be either a URL or URL
-     *     query containing the UI Bridge API JSON parameter.  The UI Bridge API JSON parameter
-     *     should contain URL-encoded JSON with two values:
-     *       - frontdoor_bridge_url
-     *       - pkce_code_verifier
-     * If pkce_code_verifier is not specified then the user agent flow is used
-     * - Returns: Boolean true if a log in attempt is possible using the provided QR code content, false
-     * otherwise
-     */
-    func loginFromQrCode(
-        loginQrCodeContent: String?
-    ) -> Bool {
-        if let uiBridgeApiParameters = uiBridgeApiParametersFromLoginQrCodeContent(
-            loginQrCodeContent
-        ) {
-            loginWithFrontdoorBridgeUrl(
-                uiBridgeApiParameters.frontdoorBridgeUrl,
-                pkceCodeVerifier: uiBridgeApiParameters.pkceCodeVerifier
-            )
-            return true
-        } else {
-            return false
-        }
-    }
-
+    
+    // MARK: - QR Code Login Via Salesforce Identity API UI Bridge Public Implementation
+    
     /**
      * Automatically log in with a UI Bridge API front door bridge URL and PKCE code verifier.
+     *
+     * This method is the intended entry point to Salesforce Mobile SDK when using the Salesforce
+     * Identity API UI Bridge front door URL.  Usable, default implementations of methods are
+     * provided for parsing the UI Bridge parameters from the reference JSON and log in URLs used
+     * by the reference QR Code Log In implementation.  However, the URL and JSON structure in the
+     * reference implementation is not required.  An app may use a custom structure so long as this
+     * entry point is used to log in with the front door URL and optional PKCE code verifier.
+     *
      * - Parameters
      *   - frontdoorBridgeUrl: The UI Bridge API front door bridge URL
-     *   - pkceCodeVerifier: The PKCE code verifier
+     *   - pkceCodeVerifier: The optional PKCE code verifier, which is not required for User Agent
+     * Authorization Flow but is required for Web Server Authorization Flow
      */
-    func loginWithFrontdoorBridgeUrl(
+    class func loginWithFrontdoorBridgeUrl(
```

Comment:
This methods do live with the login activity on Android, but I hadn't moved them all to its (static) companion object.  This was convenient for allowing the app's scene delegate to call in, but perhaps we want to scope them in a different way?  I'm open to suggestions or this works.